.github/workflows/test-suite-desktop-e2e.yml

```diff
@@ -0,0 +1,86 @@
+name: "[Build/Test] suite-desktop e2e"
+# this action builds a desktop version of Suite and runs basic test suite for it
+
+# run only if there are changes in suite or related libs paths
+on:
+  pull_request:
+    branches:
+      - develop
+    paths-ignore:
+      - "suite-native/**"
+      - "packages/connect*/**"
+      - "packages/react-native-usb/**"
+      # ignore unrelated github workflows config files
+      - ".github/workflows/connect*"
+      - ".github/workflows/suite-native*"
+      - ".github/workflows/build-desktop*"
+      - ".github/workflows/release*"
+      - ".github/workflows/template*"
+
+jobs:
+  run-desktop-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - TEST_FILE: "spawn-bridge"
+            CONTAINERS: "trezor-user-env-unix"
+          - TEST_FILE: "suite-guide"
+            CONTAINERS: "trezor-user-env-unix"
+          - TEST_FILE: "wallet-discovery"
+            CONTAINERS: "trezor-user-env-unix"
+          # - TEST_FILE: "cardano-discovery"
+          #   CONTAINERS: "trezor-user-env-unix"
+          # - TEST_FILE: "eap-modal"
+          #   CONTAINERS: "trezor-user-env-unix"
+          - TEST_FILE: "electrum"
+            CONTAINERS: "trezor-user-env-unix electrum-regtest"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+          cache: yarn
+
+      - name: Install deps and build libs
+        run: |
+          yarn install --immutable
+          yarn message-system-sign-config
+          yarn workspace @trezor/suite-data build:lib
+          yarn workspace @trezor/connect-iframe build:lib
+          yarn workspace @trezor/transport-bridge build:lib
+
+      - name: Build linux suite-desktop
+        run: |
+          yarn workspace @trezor/suite-desktop build:linux
+          bash packages/suite-desktop-core/scripts/gnupg-sign.sh
```

Comment:
is such a complicated build needed? doesn't it necessarily eat much CPU time? just app.js should be enough, isn't it?